src/m_pd.h

```diff
@@ -877,15 +877,15 @@ typedef union
     unsigned int ui;
 } t_bigorsmall32;
 
-static inline int PD_BADFLOAT(t_float f)  /* malformed float */
+PD_INLINE int PD_BADFLOAT(t_float f)  /* malformed float */
```

Comment:
yes. i thought so.
however, since i was unable to test, i kept the original macro implementation.

@Spacechild1 if you can test (or are otherwise confident), please push the cleanup to the branch.

Reply:
done!